db/seeds/teacher_transfers.rb

```diff
@@ -0,0 +1,112 @@
+require Rails.root.join("spec/support/school_transfer_helpers")
+
+module TeacherTransfersSeeder
+  extend SchoolTransferHelpers
+
+  MIN_SET_OF_TRANSFERS_SCENARIOS_PER_LP = 3
+
+  class << self
+    def seed!
+      print_seed_info("Creating teacher transfer scenarios for all Lead providers")
+
+      lead_providers.each do |lead_provider1|
+        print_seed_info("Processing Lead provider: #{lead_provider1.name}")
+
+        excluded_providers = [lead_provider1]
+        MIN_SET_OF_TRANSFERS_SCENARIOS_PER_LP.times do
+          lead_provider2 = select_different_lead_provider(excluding_lead_providers: excluded_providers)
+          create_set_of_transfers_scenarios(lead_provider1, lead_provider2)
+          excluded_providers << lead_provider2
```

Comment:
Why do we add to this list on each iteration? I would have thought we would just want to exclude the current lead provider?